tests/test_serialization.py

```diff
@@ -17,30 +17,29 @@ def test_load_multiple(tmp_path, datafile):
     tlm_list = sorted(dbload_dir.glob("*.gr"))
 
     generated_data = None
+
     for headerfile in tlm_list:
         # gather data using loadData
         hdata = loadData(headerfile, headers=True)
         data_table = loadData(headerfile)
 
         # check path extraction
-        generated_data = serialize_data(headerfile, hdata, data_table, dt_colnames=["r", "gr"], show_path=True)
+        generated_data = serialize_data(headerfile, hdata, data_table, dt_col_names=["r", "gr"], show_path=True)
```

Comment:
`df_col_names` and `df_colnames` are used across, so sticking with `dt_col_names`